core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -85,9 +86,9 @@ object ConfigCommand extends Config {
 
     configs.putAll(configsToBeAdded)
     configsToBeDeleted.foreach { config =>
-      // log an error if the config to be deleted does not exist
+      // fail the command if the config to be deleted does not exist
       if (!configs.containsKey(config))
-        System.err.println(s"Deleting config '$config' from entity '$entityName' of type '$entityType' failed, because the specified config does not exist.")
+        throw new InvalidConfigException(s"The config '$config' does not exist.")
```

Comment:
How about we collect all the configs that don't exist (via `filter` for example) and include them in the error?